modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java

```diff
@@ -357,17 +363,92 @@ public static class WithoutServices {
 
         @Test
         public void s3ServiceStartLazily() {
-            S3Client s3 = S3Client
-                .builder()
-                .endpointOverride(localstack.getEndpointOverride(Service.S3))
-                .credentialsProvider(
-                    StaticCredentialsProvider.create(
-                        AwsBasicCredentials.create(localstack.getAccessKey(), localstack.getSecretKey())
+            try (
+                S3Client s3 = S3Client
+                    .builder()
+                    .endpointOverride(localstack.getEndpointOverride(Service.S3))
+                    .credentialsProvider(
+                        StaticCredentialsProvider.create(
+                            AwsBasicCredentials.create(localstack.getAccessKey(), localstack.getSecretKey())
+                        )
                     )
+                    .region(Region.of(localstack.getRegion()))
+                    .build()
+            ) {
+                assertThat(s3.listBuckets().buckets()).as("S3 Service is started lazily").isEmpty();
+            }
+        }
+    }
+
+    public static class WithVersion2 {
+
+        private static Network network = Network.newNetwork();
+
+        @ClassRule
+        public static LocalStackContainer localstack = new LocalStackContainer(
+            DockerImageName.parse("localstack/localstack:latest")
```

Comment:
Replace by proper tag. It should be `v2.0`

```suggestion
            DockerImageName.parse("localstack/localstack:v2.0")
```